### PR TITLE
Yep 235 add tool tips

### PR DESCRIPTION
**Pull request by iittooss**

### CODE REVIEW

File: front/src/assets/i18n/en.json [hunk 1]
```diff
@@ -40,18 +40,99 @@
...
+      "addPicture": "Add a profile picture"
+    },
+    "genreStep": {
+      "label": "What are your styles ?",
+      "max": "You can select up to 3 styles",
+      "helper": "What kind of music do you make?"
+    },
+    "socialLinksStep": {
+      "label": "Where can we find you ?",
+      "links": "Fill out links you want people see",
```
Comment (filveith) [inline front/src/assets/i18n/en.json:122]:
```suggestion
      "links": "Fill out links you want people to see",
```

File: front/src/assets/i18n/en.json [hunk 2]
```diff
@@ -40,18 +40,99 @@
...
+    "presentationPicturesStep": {
+      "label": "Express yourself in pictures!",
+      "helper": "Everyone will bnameHelpere able to recognize you with this pictures",
+      "add": "Add pictures",
+      "max": "Add up to 5 pictures"
+    },
+    "capacityStep": {
+      "label": "How many people can you accommodate?",
+      "capacity": "How many people can you accommodate?",
+      "helper": "Choose the capacity which is higher than your actual capacity. If you can accommodate 30 people, choose 50 max."
```
Comment (filveith) [inline front/src/assets/i18n/en.json:134]: We really need to edit this capacity stuff, this is not ideal, but anyway this is for another time

File: front/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -40,18 +40,100 @@
...
-    "socialLinksStep": "Où êtes vous présent ?",
-    "addressCompleteStep": "Où êtes vous présent ?",
-    "genreStep": "Vous êtes plutôt ?",
-    "presentationPicturesStep": "À quoi ressemble votre groupe ?",
-    "profilePictureStep": "À quoi vous ressemblez ?",
-    "capacityStep": "Vous pouvez acceuillir combien de personne ?"
+    "profileTypeStep": {
+      "label": "Vous êtes ?",
+      "artist": "Un artiste",
+      "artistTooltip": "Chanteur, Groupe, Musicien, Formation",
```
Comment (filveith) [inline front/src/assets/i18n/fr.json:49]: What is Formation ?
```suggestion
      "artistTooltip": "Chanteur, Groupe, Musicien",
```

File: front/src/assets/i18n/fr.json [hunk 4]
```diff
@@ -40,18 +40,100 @@
...
+      "host": "Un hôte",
+      "hostTooltip": "Bar, Pub, Restaurant, Salle de Concert"
+    },
+    "mailPhoneStep": {
+      "label": "Pour vous retrouver",
+      "email": "Email",
+      "emailHelper": "Un email pour communiquer avec vous",
+      "phone": "Numéro de Téléphone",
```
Comment (filveith) [inline front/src/assets/i18n/fr.json:57]:
```suggestion
      "phone": "Numéro de téléphone",
```

**Review by filveith — APPROVED** [review]

File: front/src/assets/i18n/en.json (hunk 2 above)
Comment (iittooss) [reply front/src/assets/i18n/en.json:134]: :zap: :+1: :zap:

**Pull request merged**